### DISCUSSION TITLE
How to set prefix url "/auth" in version17+ just like the old version

### DISCUSSION THREAD

**Discussion by lubw7** [keycloak/keycloak#24232] [Q&A]
### Before reporting an issue

- [X] I have searched existing issues
- [x] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

docs

### Describe the bug

for example, in version 18.0.2, get token by url /realms/{relmname}/protocol/openid-connect/token
in old version, we get token by url **/auth**/realms/{relmname}/protocol/openid-connect/token
the old project just support url that bring "/auth"
how to set prefix "/auth" in new version , run keycloak by podman/docker

### Version

18.0.2

### Expected behavior

use  **/auth**/realms/{relmname}/protocol/openid-connect/token get token

### Actual behavior

 use /realms/{relmname}/protocol/openid-connect/token

### How to Reproduce?

as mentioned above

**Comment by Xulunix** [comment]
Setting https://www.keycloak.org/server/all-config?q=http-relative-path to "/auth" results in the desired pre-18.0 URLs
